pages/polyfills.js

```diff
@@ -0,0 +1,6 @@
+// Use the same helper as Babel to avoid bundle bloat.
+import 'core-js/modules/es6.array.find-index';
+import 'core-js/modules/es6.map';
+import 'core-js/modules/es6.set';
```

Comment:
This should've been there in the first place: https://reactjs.org/docs/javascript-environment-requirements.html